Sources/PackageDescription/SupportedPlatforms.swift

```diff
@@ -21,6 +20,11 @@ public struct Platform: Encodable, Equatable {
         self.name = name
     }
 
+    /// Create a custom platform.
```

Comment:
The convention is using 3rd person present tense, so "Creates" instead of "Create".